addons/mail/static/src/core/common/message.scss

```diff
@@ -158,18 +158,6 @@
     z-index: $o-mail-NavigableList-zIndex;
 }
 
-.o-mail-Message-openActionMobile {
-    opacity: 10% !important;
-
-    .o-mail-Message.o-card & {
-        opacity: 15% !important;
-    }
-
-    &:active {
-        opacity: 75% !important;
-    }
-} 
-
```

Comment:
Ok to remove it completely. Hopefully people won't be scared and will try long-press.
I wonder if we shouldn't still keep it? Not a big deal.